scripts/db/drop_schema.sql

```diff
@@ -109,10 +109,10 @@ DROP INDEX IF EXISTS asset_users_user_id_idx;
 
 --
 
-DROP SEQUENCE IF EXISTS public.activities_id_seq;
 DROP TABLE IF EXISTS public.activities;
-DROP SEQUENCE IF EXISTS public.activity_types_id_seq;
+DROP SEQUENCE IF EXISTS public.activities_id_seq;
 DROP TABLE IF EXISTS public.activity_types;
+DROP SEQUENCE IF EXISTS public.activity_types_id_seq;
```

Comment:
Small fix to order of DROPs

Reply:
@johncrossman Whoops! I think https://github.com/ets-berkeley-edu/squiggy/pull/15 fixes this.

Reply:
Thanks @pkerschen.  I'll close this and rebase.